src/service/channel-service.js

```diff
@@ -32,81 +60,122 @@ var makeName = function (channelName, topic) {
- * @param  {Object}   context  Context in which the callback will be executed
- * @param  {Object}   options  Object with defaults
- * @param  {Number}   options.priority  Can be used to control order of operations, defaults to 0, can be any +ve or -ve number
- * @param  {String|Number|Function}   options.value  Callback will only be triggered if this condition matches. See Examples for details
- * @return {string}            Returns a token you can later use to un-subscribe
- *
- *  **Examples**
- *  subscribe('run', cb) - subscribe to changes on a top-level 'run' topic
- *  subscribe('run/*') - changes on children of the run object. Note this not also be triggered for changes to run.x.y.z
- *  subscribe(['run', 'run/*']) - changes to run and children
- *  subscribe('run/variables/price', cb) - changes to price
- *
- *  subscribe('run', function () { this.innerHTML = 'Triggered'}, document.body); - set the context for the callback
- *  subscribe('run', cb, this, {priority: 9}); - by default priority is 0. Can be used to control order of operations
- *  subscribe('run/variables/price', cb, this, {priority: 30, value: 50}); - only exec callback if value of price is 50
- *  subscribe('run/variables/price', cb, this, {priority: 30, value: '>50'}); - only exec callback if value of price is > 50
- *  subscribe('run/variables/price', cb, this, {priority: 30, value: function (val) {return val % 2 ===0}}); - only exec callback if value of price is even
- *
- */
-Channel.prototype.subscribe = function (topic, callback, context, options) {
-    var topics = [].concat(topic);
-    var me = this;
-    var subscriptionIds = [];
-    var opts = me.channelOptions;
-
-    opts.transport.batch(function () {
-        $.each(topics, function (index, topic) {
-            topic = makeName(opts.base, opts.topicResolver(topic));
-            subscriptionIds.push(opts.transport.subscribe(topic, callback));
+
+Channel.prototype = $.extend(Channel.prototype, {
+
+    /**
+     * Subscribe to changes on a topic.
+     *
+     *  **Examples**
+     *
+     *      // Subscribe to changes on a top-level 'run' topic
+     *      cs.subscribe('run', cb);
+     *
+     *      // Subscribe to changes on children of the 'run' topic. Note this not also be triggered for changes to run.x.y.z.
+     *      cs.subscribe('run/*');
+     *
+     *      // Subscribe to changes on both the top-level 'run' topic and its children
+     *      cs.subscribe(['run', 'run/*']);
+     *
+     *      // Subscribe to changes on a particular variable
+     *      subscribe('run/variables/price', cb);
+     *
+     *      // Set the context for the callback
+     *      cs.subscribe('run', function () { this.innerHTML = 'Triggered'}, document.body);
+     *
+     *      // Control the order of operations by setting the `priority`
+     *      cs.subscribe('run', cb, this, {priority: 9});
+     *
+     *      // Only execute the callback, `cb`, if the value of the `price` variable is 50
```

Comment:
no. but you can change to // and then cut/paste to above the /* */ comment block and they'll be hidden.